cpp/src/IceGrid/FileCache.cpp

```diff
@@ -4,7 +4,7 @@
 #include "../Ice/FileUtil.h"
 #include "Ice/Communicator.h"
 #include "Ice/Properties.h"
-#include "IceGrid/Exception.h"
```

Comment:
I had to move some exceptions from `Exception.ice` to `Admin.ice` to fix some circular `@see` references.
You'll see the swap further down in this PR.

Reply:
We don't allow forward declaring exceptions, only classes & exceptions can be forward declared in Slice.